### DISCUSSION TITLE
Latest Magento 2 doesn't work in "warden debug" due to Segmentation fault.

### DISCUSSION THREAD

**Discussion by zhooravlik** [wardenenv/warden#799] [Help & Support]
Latest Magento 2.4.7 doesn't work in "warden debug" mode. Any bin/magento command only showing "Segmentation fault".
In "warden shell" it is working properly.
I've tried to remove and setup project from scratch - result is the same.

Did you face such an issue? And how to fix this?

**Comment by iranimij** [comment]
@zhooravlik I also have the same error, Did you find any solution?

**Comment by ihor-sviziev** [comment]
@zhooravlik @iranimij, the issue was discussed in https://github.com/wardenenv/warden/issues/843 and https://github.com/wardenenv/images/issues/35.
It's caused by using xdebug together with ioncube loader. As a workaround, you can disable ioncube loader in the debug container, here are more details https://github.com/wardenenv/warden/issues/843#issuecomment-2786465539
